js_modules/dagit/packages/ui/tsconfig.build.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig",
+  "compilerOptions": {
+    "outDir": "lib",
+    "noEmit": false,
+    "declaration": true,
+    "emitDeclarationOnly": true
```

Comment:
Hmm,  does rollup do the transpiling of the TS into es5? My guess is it does but I'm not familiar. Otherwise it seems like this would result in just type files and no minified source, etc?